Automattic/pocket-casts-android
# Revert media3 to 1.0.2

## Description

This reverts `media3` to `1.0.2`.

Temporarily fixes: https://github.com/Automattic/pocket-casts-android/issues/2004

## Testing Instructions

1. Listen to a podcast like this: https://pca.st/bprwy2b9
2. Set the playback speed to 1.9x and set Trim Silence to Mad Max
3. Play the episode and notice that it ends properly

Important: Smoke test playback on Watch. 

## Checklist
 
#### I have tested any UI changes...
